lib/components/app/app-menu.tsx

```diff
@@ -238,57 +238,71 @@ class AppMenu extends Component<
               className="app-menu-trip-planner-link"
               icon={<MapMarked />}
               onClick={this._showTripPlanner}
-              text={
-                <FormattedMessage id="components.BatchRoutingPanel.shortTitle" />
-              }
+              text={intl.formatMessage({
+                id: 'components.BatchRoutingPanel.shortTitle'
+              })}
             />
             {/* This item is duplicated by the view-switcher, but only shown on mobile
             when the view switcher isn't shown (using css) */}
             <AppMenuItem
               className="app-menu-route-viewer-link"
               icon={<Bus />}
               onClick={this._showRouteViewer}
-              text={<FormattedMessage id="components.RouteViewer.shortTitle" />}
+              text={intl.formatMessage({
+                id: 'components.RouteViewer.shortTitle'
+              })}
             />
             {/* This item is duplicated by the view-switcher, but only shown on mobile
             when the view switcher isn't shown (using css) */}
             <AppMenuItem
               className="app-menu-route-viewer-link"
               icon={<MapPin />}
               onClick={this._showNearby}
-              text={<FormattedMessage id="components.ViewSwitcher.nearby" />}
+              text={intl.formatMessage({
+                id: 'components.ViewSwitcher.nearby'
+              })}
             />
             <AppMenuItem
               icon={<Undo />}
               onClick={this._startOver}
-              text={<FormattedMessage id="common.forms.startOver" />}
+              text={intl.formatMessage({
+                id: 'common.forms.startOver'
+              })}
             />
             {popupTarget && (
               <AppMenuItem
                 icon={<SvgIcon iconName={popupTarget} />}
                 onClick={this._triggerPopup}
-                text={<PopupTriggerText popupTarget={popupTarget} />}
+                text={intl.formatMessage({
+                  id: `config.popups.${popupTarget}`
+                })}
```

Comment:
Why are we not using `PopupTriggerText` anymore?

Reply:
That was a mistake! I will fix